pkg/arvo/lib/group-store.hoon

```diff
@@ -0,0 +1,455 @@
+/-  *group, sur=group-store
+/+  resource
+^?
+=<  [. sur]
+=,  sur
+|%
+::
+++  dekebab
```

Comment:
>different names between hoon and json

I get it, kebab case is a pain to work with in js, but...  
At least this isn't doing them all by hand. (^: